payment-request/allowpaymentrequest/setting-allowpaymentrequest-timing.https.sub.html

```diff
@@ -18,7 +18,8 @@
   });
 
   window.onmessage = t.step_func_done((e) => {
-    assert_equals(e.data.message, 'Success');
+    assert_equals(e.data.message, 'Exception');
```

Comment:
The test is inconsistent with its description at line 7 and line 8.

Reply:
It's not, actually, but I have expanded on the comment to explain in more detail what's going on.